Don't install JPLEPH, we symlink to the same file in wsjtx-data.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1124,11 +1124,11 @@ install (FILES
   #COMPONENT runtime
   )
 
-install (FILES
-  contrib/Ephemeris/JPLEPH
-  DESTINATION ${CMAKE_INSTALL_DATADIR}/${CMAKE_PROJECT_NAME}
-  #COMPONENT runtime
-  )
+#install (FILES
+#  contrib/Ephemeris/JPLEPH
+#  DESTINATION ${CMAKE_INSTALL_DATADIR}/${CMAKE_PROJECT_NAME}
+#  #COMPONENT runtime
+#  )
 
 #
 # Mac installer files
